scrunch/tests/conftest.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+import pytest
+
+mark_fail_py2 = pytest.mark.xfail(sys.version_info < (3, 0,), reason="py2 order in args causes tests failures")
```

Comment:
I wanted to use this one instead of duplicating the one that I used for the new integration tests. I'll refactor it. Thanks

Reply:
So this is inside the conftest file and I'm using in some tests that were failing when running tox with py2.7.
I run all the tests with py27 locally and they pass, but with tox we have failures. Since we are running tox during the PR, I just wanted not to be blocked by some issue with a deprecated version of python